wile/cert.py

```diff
@@ -62,9 +64,16 @@ def cert():
                       for one day. Supported units are hours, days and weeks.''')
 @click.option('--force', is_flag=True, default=False, show_default=True,
               help='Whether to force a request to be made, even if a valid certificate is found')
+@click.option('--ssh-private-key',
+              type=click.Path(file_okay=True, dir_okay=False),
+              default=None, show_default=True, help='Path to SSH private key')
+@click.option('--ssh-private-key-pass', default=None, envvar='WILE_SSH_PASS',
+              help='SSH private key password')
```

Comment:
Sorry, I probably didn't explain my point well the first time: I don't think it's a good idea to have this as an option _at all_. This encourages bad security practices of passing passwords on command-line. The passphrase should be directly read from the environment and the connection should just fail if it's wrong.
Remember the user still has the option of running an ssh-agent (which has other advantages), or running wile with:
```
$ WILE_SSH_PASS=somepass wile cert ...
```

We should just make sure this is mentioned in the help.